lib/robotis/control_table/tables.ex

```diff
@@ -136,6 +136,139 @@ defmodule Robotis.ControlTable.Tables do
     indirect_data_20: {227, 1, 1}
   }
 
+  # XM430 series (W210, W350) - Protocol 2.0
+  # Control table is nearly identical to XL330, same addresses and scaling
+  # Main differences: model number, current limit unit (2.69mA vs 1mA)
+  @xm430 %{
```

Comment:
n.b. We might want to split these into different files just to make it more manageable.